data/src/main/java/me/androidbox/data/local/converter/EventConverter.kt

```diff
@@ -0,0 +1,25 @@
+package me.androidbox.data.local.converter
+
+import androidx.room.TypeConverter
+import com.squareup.moshi.Moshi
+import com.squareup.moshi.kotlin.reflect.KotlinJsonAdapterFactory
+import me.androidbox.data.remote.model.request.EventCreateRequestDto
+import javax.inject.Inject
+
+class EventConverter @Inject constructor(){
```

Comment:
If you have an @Inject constructor, you don't need a @Provides function for this class